<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaSetup</class>
 <widget class="QWidget" name="TelaSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelOP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>35</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>28</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>81</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tempo:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditTempo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Iniciar Produção</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
